serverless.yml

```diff
@@ -38,16 +44,21 @@ custom:
   observationDb:
     connectInfo: ${ssm:/aws/reference/secretsmanager/WQP-EXTERNAL-${self:provider.stage}~true}
   bucketName: iow-rdb-files
+  mailingList: ${ssm:/aws/reference/secretsmanager/AQTS-CAPTURE-MAILING-LISTS~true}
 
 functions:
   loadRdb:
     handler: gov.usgs.wma.waterdata.groundwater.BuildRdbFileHandler
-    role: arn:aws:iam::${self:custom.accountNumber}:role/csr-Lambda-Role
+    role:
+      Fn::Sub:
+        - arn:aws:iam::${accountId}:role/csr-Lambda-Role
+        - accountId:
+            Ref: AWS::AccountId
     events:
       - schedule:
           name: etl-discrete-groundwater-rdb-${self:provider.stage}-event-all
           description: "Schedule event to generate RDB file by invoking etl-discrete-groundwater-rdb lambda"
-          rate: cron(0 13 * * ? *)
+          rate: cron(0 14 * * ? *)
```

Comment:
13 was too close to the start time of the obs db on test - basically this lambda was failing most of the time which could have contributed to some weird errant data behavior on the NWISWeb side for a little while until we manually ran it again.